crates/lib/src/qpu/client.rs

```diff
@@ -165,16 +164,16 @@ impl Qcs {
         let target = gateways
             .first()
             .ok_or_else(|| GrpcEndpointError::NoEndpoint(quantum_processor_id.to_string()))?;
-        parse_uri(&target.url).map_err(GrpcEndpointError::BadUri)
+        parse_uri(&target.url).map_err(GrpcEndpointError::GrpcError)
     }
 }
 
 /// Errors that may occur while trying to resolve a `gRPC` endpoint
 #[derive(Debug, thiserror::Error)]
 pub enum GrpcEndpointError {
-    /// Error due to a malformed URI
-    #[error("Malformed URI for endpoint: {0}")]
-    BadUri(#[from] GrpcError<RefreshError>),
+    /// Error due to a bad gRPC configuration
+    #[error("Error configuring gRPC request: {0}")]
+    GrpcError(#[from] GrpcError<RefreshError>),
```

Comment:
FYI while this rename looks good, it is also a breaking change, make sure the commit message reflects that

Reply:
Sure thing, changed commit to `feat!:`